packages/visx-annotation/src/components/Label.tsx

```diff
@@ -255,3 +322,39 @@ export default function Label({
     </Group>
   );
 }
+
+interface AnchorLineProps {
+  anchorLineOrientation: 'horizontal' | 'vertical';
+  verticalAnchor: TextProps['verticalAnchor'];
+  horizontalAnchor: TextProps['textAnchor'];
+  anchorLineStroke: string;
+  width: number;
+  height: number;
+}
+function AnchorLine({
```

Comment:
thanks for pulling this out 🙏  . if we pull `HtmlLabel` into a separate component, might have to do the same with this so it can be imported by `HtmlLabel` (don't think we need to add this to the docs/export from the package index, tho)